### PR TITLE
Log models and scripts before run start

### PR DESCRIPTION
**Pull request by mkam039**

### CODE REVIEW

File: src/fal/cli.py [hunk 1]
```diff
@@ -50,7 +50,15 @@ def run(run, project_dir, profiles_dir, keyword, all, debug):
...
-        ordered_scripts = ScriptGraph(models, keyword, project_dir).sort()
+
+        models_arr = map(
+            lambda model: f"{model.name}: {', '.join(model.meta['fal']['scripts'])}",
+            models
+        )
+        models_str = '\n'.join(models_arr)
+        print_timestamped_line(f"Starting FAL run for following models (model_name: scripts): \n{models_str}")
+
+        # ordered_scripts = ScriptGraph(models, keyword, project_dir).sort()
```
Comment (mkam039) [inline src/fal/cli.py:61]: this breaks on a schema.yml with multiple scripts, such as:

```
version: 2

models:
  - name: boston
    description: Ozone levels
    config:
      materialized: table
    meta:
      fal:
        scripts:
          - fal_scripts/slack.py
  - name: stg_zendesk_ticket_data
    description: zendesk ticket data
    config:
      materialized: table
    meta:
      fal:
        scripts:
          - fal_scripts/zendesk_sentiment_analysis.py
  - name: zendesk_ticket_metrics
    description: Zendesk ticket metrics
    config:
      materialized: table
    meta:
      fal:
        scripts:
          - fal_scripts/forecast_slack.py
```

with this error:

```
Exception: Your python scripts contain a cycle could not determine the right order
```

Comment (mkam039) [reply src/fal/cli.py:61]: @turbo1912

File: src/fal/cli.py [hunk 2]
```diff
@@ -50,7 +50,15 @@ def run(run, project_dir, profiles_dir, keyword, all, debug):
...
         project = parse_project(real_project_dir, real_profiles_dir, keyword)
         models = project.get_filtered_models(all)
-        ordered_scripts = ScriptGraph(models, keyword, project_dir).sort()
+
+        models_arr = map(
+            lambda model: f"{model.name}: {', '.join(model.meta['fal']['scripts'])}",
+            models
+        )
+        models_str = '\n'.join(models_arr)
+        print_timestamped_line(f"Starting FAL run for following models (model_name: scripts): \n{models_str}")
```
Comment (turbo1912) [inline src/fal/cli.py:59]: can you move this its own function and outside of this file, we keep adding more logic here and trying to keep cli.py interface to the cli

**Review by chamini2 — APPROVED** [review]
Looks good!

**Pull request merged**